modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the YAML view
+
+.Prerequisites
+
+* You must have administrator privileges.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Administration* -> *Cluster Settings*.
+. Select the *Configuration* tab, click the *Console (operator.openshift.io)* resource and see the *Details* page.
+. Click the *YAML* tab to open the editor and edit it as per your requirements. 
```

Comment:
This is potential fluff: "and edit it as per your requirements." I think you can get rid of it completely or rephrase it a bit.

Reply:
Would changing it to "edit the code snippet" be the correct rephrased text?

Reply:
Try this instead:

Click the *YAML* tab to open the editor and edit the YAML content as needed.

Reply:
Thank you for the suggestion. Incorporating it as you suggested.